services/wordpress/wordpress-base.js

```diff
@@ -25,16 +25,14 @@ module.exports = class BaseWordpress extends BaseJsonService {
     throw new Error(`extensionType() function not implemented for ${this.name}`)
```

Comment:
is this still necessary? I _think_ I saw it removed from all the base classes

Reply:
Ah, you're correct, it can be removed.